ResGen/ResGen.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Generates C# typed bindings for .resx files</Description>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>resgen</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifiers>win7-x86;win7-x64;osx.10.12-x64;linux-x64</RuntimeIdentifiers>
```

Comment:
Why do we need the RIDs if the ResGen tool runs only locally on the build machine?

Reply:
I just _borrowed_ this from the PSCore repo, didn't really change anything except the `program.cs` file

Reply:
also, I really like the `Target` in Engine.csproj, I'll change to use that rather than the explicit `-ResGen` switch.